lang/python/python3-pyinotify/Makefile

```
@@ -0,0 +1,37 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-pyinotify
+PKG_VERSION:=0.9.6
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=COPYING
+
+PYPI_NAME:=pyinotify
+PKG_HASH:=9c998a5d7606ca835065cdabc013ae6c66eb9ea76a00a1e3bc6e0cfe2b4f71f4
+
+$(call include_mk, pypi.mk)
+include $(INCLUDE_DIR)/package.mk
+$(call include_mk, python3-package.mk)
```

In my suggestion or either in [this Python's README](https://github.com/openwrt/packages/blob/master/lang/python/README.md) how you can build Python package and add dependencies, there is not something like this.

```suggestion
include ../pypi.mk
include $(INCLUDE_DIR)/package.mk
include ../python3-package.mk
```

---

understood : used : `make package/python3-pyinotify/{clean,configure} PY3=stdlib V=s` and add dependencies ! thanks...